Gems/Atom/RPI/Code/Source/RPI.Public/Pass/RasterPass.cpp

```diff
@@ -211,7 +213,7 @@ namespace AZ
         void RasterPass::SetupFrameGraphDependencies(RHI::FrameGraphInterface frameGraph)
         {
             RenderPass::SetupFrameGraphDependencies(frameGraph);
-            frameGraph.SetEstimatedItemCount(static_cast<uint32_t>(m_drawListView.size()));
+            frameGraph.SetEstimatedItemCount(static_cast<uint32_t>(m_drawListView.size() * AZStd::thread::hardware_concurrency()));
```

Comment:
I think this will not work. If you change this code you will also need to update GetDrawListPartition to reflect this change. I think itemsPerPartition is incorrect now. 

Maybe a better and more simpler approach to do what you are trying to do is by reducing the value of m_commandListCostThresholdMin. The only difference is that this approach will have impact across all passes. Reducing this number will increase the number of threads used by scopes for the same ItemCost. You could change the hardcoded value or you could set it in the setreg file as this value can also be data driven per project if needed.

Reply:
m_commandListCostThresholdMin tells RHI the minimum amount of work one thread/commandlist can do, so if you reduce this number you are essentially increasing the number of threads needed and will go wide'r' for all scopes.

Reply:
This doesn't look right, SetEstimatedItemCount is suppose to help the RHI know roughly how many draw items will be passed so it can schedule the work properly, and m_drawListView.size() is the exact amount, so the code should be ideal as is. I don't know why multiplying by hardware_concurrency would make things faster, but to my knowledge this should imply something is wrong in the RHI, no? Either way it doesn't seem like this is a sound fix.
If there is some reason that I'm missing why this is actually a correct and logical fix at least put a comment detailing the reasoning behind it, since someone without context looking at the code will have know idea where that multiplication comes from.

Reply:
Increasing SetEstimatedItemCount increases the number of contexts (i.e threads) but you also need to partition the work to accommodate for that. With this change we are basically saying that this scope has **more** work so give it more threads but we are not accommodating the work given to each context accordingly. I have recommended a better approach to get this behavior above.